<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="queryLine">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Input query</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="search_btn">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="UDB_btn">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Update document base</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="resultList"/>
        </item>
        <item>
         <widget class="QPushButton" name="show_more_btn">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Show all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTextBrowser" name="servMessages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="markdown">
       <string/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="servMesagesButtons">
      <item>
       <widget class="QPushButton" name="servMess_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Service messages</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="servMessagesClear_btn">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Clear service messages</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>servMess_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sm_hideShow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servMessagesClear_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sm_clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UDB_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateDocBase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_more_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>692</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>find()</slot>
  <slot>updateDocBase()</slot>
  <slot>showMore()</slot>
  <slot>sm_hideShow()</slot>
  <slot>sm_clear()</slot>
 </slots>
</ui>
